shared/package.json

```diff
@@ -272,12 +272,12 @@
     "msgpack": "git://github.com/keybase/nullModule",
     "net": "git://github.com/keybase/nullModule",
     "null-loader": "0.1.1",
-    "pegjs": "0.10.0",
     "prettier": "1.17.1",
     "randexp": "0.5.3",
     "react-hot-loader": "4.11.0",
     "react-test-renderer": "16.8.4",
     "redux-logger": "3.0.6",
+    "rimraf": "3.0.0",
```

Comment:
just use 2.2.8 so we don't have any new code

Reply:
Making it 2.6.3 which we also already have makes fewer lockfile changes, i'll use that one